### ISSUE TITLE
✍️ Contribution period: Shamika Anne Sawalha

### ISSUE THREAD

**Issue by luckyst4rdolph1n** [ersilia-os/ersilia#1045]
### Week 1 - Get to know the community

- [X] Join the communication channels
- [X] Open a GitHub issue (this one!)
- [X] Install the Ersilia Model Hub and test the simplest model
- [X] Install Docker if needed, and test another model
- [x] Write a motivation statement to work at Ersilia
- [x] Submit your first contribution to the Outreachy site

### Week 2 - Get Familiar with Machine Learning for Chemistry

- [x] Select a model from the list suggested in GitBook
- [x] Download and serve the model via the Ersilia Model Hub to ensure it works
- [ ] Open a repository on your GitHub user with all the necessary files
- [ ] Select and clean a dataset of 1000 molecules (example notebook 1)
- [ ] Run predictions for the molecules on the selected model and evaluate the results

### Week 3 - Validate a Model in the Wild

- [ ] Find a suitable dataset with sufficient experimental results
- [ ] Clean and standardize the dataset
- [ ] Run predictions and calculate metrics.

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by luckyst4rdolph1n** [comment]
With a profound passion for both technology and healthcare, I am eager to contribute my skills and knowledge to Ersilia's cause which aligns seamlessly with my aspiration to make technology equally accessible and useful across healthcare industries.

Throughout my academic journey and hands-on experiences, I have gained a solid foundation in programming with Python, creating useful projects and fun games alike. With my continuous learning, I discovered Python's application in machine learning more particularly its capacity for data handling and analysis. As an inquisitive person, this prompted me to experiment with its features, learning about its implementation across discipline and indsutries including healthcare. The revolutionary potential of these technologies in tackling difficult healthcare issues intrigues me. Helping to make a significant difference in global health is a goal of mine, and Ersilia's dedication to using AI and ML to improve research capacities for infectious and neglected diseases resonates deeply with this goal.

To me, Ersilia stands out for its dedication to open-source values in addition to its technological focus. The culture of volunteerism and inclusion within the community proves to me that change and advancement can come from anyone. My principles are completely aligned with the concept of community-driven, transparent, and cooperative solutions. I'm excited to add to a culture that values creativity, diversity, and our common objective of enhancing the health of disadvantaged groups.

Throughout my previous experiences, I have effectively used machine learning techniques to tackle practical issues, and I am sure that my technical expertise and enthusiasm for healthcare will make me an invaluable addition to Ersilia. Working on initiatives that directly affect public health and assisting in the creation and application of innovative instruments with the potential to save lives keeps me going.

As an applicant and hopefully a future intern, I am excited to pick up knowledge from the bright and committed staff members at Ersilia and get a deeper understanding of the relationship between AI, ML, and medical research. I have no doubt that this experience will improve my abilities and make a big difference in the Ersilia's continued mission. Thank you for considering my application, and I am looking forward to making a difference with you all.
